### DISCUSSION TITLE
Support for Kelvin in Color Temperature Selector

### DISCUSSION THREAD

**Discussion by schelv** [home-assistant/frontend#16958] [Other feature requests]
I recommend adding the Kelvin scale to the Home Assistant's [Color Temperature Selector](https://www.home-assistant.io/docs/blueprint/selectors/#color-temperature-selector).
Currently it only supports mireds. 
This is not convenient when you want Kelvin as the color temperature as you have to convert between kelvins and mireds in multiple places.

**Comment by karwosts** [comment]
When you say "add the kelvin scale", what do you mean exactly?

Do you want a yaml option when instancing a selector that changes the display between kelvin and mired scale? I.e. End users of the selector will not be able to choose, but I'm imagining this is a choice of the developer who is the author of a blueprint or script. Is that what you're asking for?

**Comment by spacegaier** [comment]
I think you are talking about this, which I had started on some time ago? https://github.com/home-assistant/frontend/pull/14495

I need to rework that a bit though to not break old blueprints.

**Comment by schelv** [comment]
Here you can see the problem I'm facing with the current color temperature selector: https://github.com/home-assistant/core/pull/94394/files
I want a color select thingy, but the thingy returns mireds.
Which is annoying because I don't want mireds.

**Comment by schelv** [comment]
@spacegaier if I can help in any way, let me know.👍
